src/main/java/com/fundacionjala/movies/MovieChildren.java

```diff
@@ -0,0 +1,29 @@
+package com.fundacionjala.movies;
+
+/**
+ * Created by Charito on 6/10/2016.
+ */
+public class MovieChildren extends Movie {
+
+    public MovieChildren(String title) {
+        super(title);
+        super.priceRent = 1.5;
+        super.daysAllowed = 3;
+        super.priceDelay = 1.5;
+        super.points = 1;
+    }
+
```

Comment:
If you notice all those variables are needed in the child classes
So, I suggested you to put them into the constructor class
and have constants variable for those numbers in the child classes.